src/administration/Administration.Service/Controllers/DocumentsController.cs

```diff
@@ -84,10 +84,8 @@ public async Task<ActionResult> GetDocumentContentFileAsync([FromRoute] Guid doc
     /// <response code="404">The document was not found.</response>
     [HttpGet]
     [Route("selfDescription/{documentId}")]
```

Comment:
```suggestion
    [Route("selfDescription/{documentId}")]
    [AllowAnonymous]
```
You forgot to add this flag here. Please also test this again.